client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -249,7 +249,8 @@ class PreviewFrame extends React.Component {
     let jsFileStrings = content.match(STRING_REGEX);
     jsFileStrings = jsFileStrings || [];
     jsFileStrings.forEach((jsFileString) => {
-      if (jsFileString.match(MEDIA_FILE_REGEX)) {
+      console.log(this.props.setBlobUrl);
+      /* if (jsFileString.match(MEDIA_FILE_REGEX)) {
```

Comment:
I think it's still a good idea to serve as many files as possible from a blob url rather than server side—this will keep AWS costs down 😄